### ISSUE TITLE
How do I change the dataset folder pattern

### ISSUE THREAD

**Issue by SurbhiKhushu** [junyanz/pytorch-CycleGAN-and-pix2pix#1118]
Hello,

I have different subjects upon which I want to train and test the model.For example , I want to have two folders train and test and inside each folder I would like to have this pattern  train>Subject1>Dataset A, Dataset B. This pattern for every subject in train and test. Where can I modify that part . Is it the image_folder.py?

Thanks

**Comment by junyanz** [comment]
Yes, you need to modify the data loader. If you use CycleGAN, please modify this [one](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/data/unaligned_dataset.py). For pix2pix, change this [one](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/data/aligned_dataset.py)
